### PR TITLE
Form showValidationErrors Support for NxFileUpload - RSC-1163

### PR DESCRIPTION
**Pull request by cindyko226**
Add Form showValidationErrors support to NxFileUpload: https://issues.sonatype.org/browse/RSC-1163

### CODE REVIEW

**Comment by rpokorny** [comment]
The `nx-file-upload__validation-error` class should no longer be used, on this feature branch there is now a unified class for these validation errors across all types of form fields: `nx-field-validation-message`

File: lib/src/components/NxFileUpload/__tests__/NxFileUpload.test.tsx [hunk 1]
```diff
@@ -38,14 +40,25 @@ describe('NxFileUpload', function() {
...
 
+  it('shows an error when in a form with showValidationErrors if it is isRequired and no file is selected', function() {
+    const renderWithForm = render(
+      <NxForm onSubmit={() => {}} showValidationErrors={true}>
+        <NxFileUpload { ...minimalProps } isRequired={true} />
+      </NxForm>
+    );
+
+    expect(renderWithForm.queryByRole('alert')).toBeTruthy();
+    expect(renderWithForm.queryByRole('alert')).toHaveTextContent('This field is Required!');
```
Comment (rpokorny) [inline lib/src/components/NxFileUpload/__tests__/NxFileUpload.test.tsx:59]: Also check `aria-invalid` and `aria-errormessage`

Comment (cindyko226) [reply lib/src/components/NxFileUpload/__tests__/NxFileUpload.test.tsx:59]: Updated here 45409a16cdfa84a6936a486b465972c6da8d9f39.

**Comment by cindyko226** [comment]
> The nx-file-upload__validation-error class should no longer be used, on this feature branch there is now a unified class for these validation errors across all types of form fields: nx-field-validation-message

Updated here 45409a16cdfa84a6936a486b465972c6da8d9f39.

File: lib/src/components/NxFileUpload/__tests__/NxFileUpload.test.tsx [hunk 2]
```diff
@@ -38,14 +40,27 @@ describe('NxFileUpload', function() {
...
+    expect(renderWithForm.container.querySelector('input[type=file]')?.getAttribute('aria-invalid')).toBe('true');
```
Comment (rpokorny) [inline lib/src/components/NxFileUpload/__tests__/NxFileUpload.test.tsx:60]:
```suggestion
    expect(renderWithForm.container.querySelector('input[type=file]')).toHaveAttribute('aria-invalid', 'true');
```

**Review by rpokorny — APPROVED** [review]

**Pull request merged**